src/pages/pet/pet.module.scss

```diff
@@ -43,3 +44,44 @@
 .addPetBtn {
   margin-left: 20px;
 }
+
+.responsiveContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.responsiveItem {
+  margin: 0 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+@media (min-width: 476px) {
+  .dogPic {
+    height: 400px;
+  }
+}
+
+@media (min-width: 768px) {
+  .responsiveContainer {
+    flex-direction: row;
+  }
+  .dogPic {
+    width: 60vw;
+    height: 450px;
+    padding-left: 20px;
+  }
+  .responsiveItem {
+    order: 2;
+    margin-left: 40px;
+    margin-right: 20px;
+    flex: 1;
+    .sectionHeader {
+      padding-top: 0;
+    }
+  }
+  .addContainer{
+    margin-top: 45px;
+    order: 2;
+  }
+}
```

Comment:
nit: Don't forget to add empty line to the end of the file

Reply:
+